news/10004.trivial.rst

```diff
@@ -0,0 +1 @@
+- Annotate "typing.List" into "tools.tox_pip.pip()" (http://github.com/pypa/pip/pull/10004)
```

Comment:
```suggestion
Annotate `typing.List` into `tools.tox_pip.pip()`
```

You don’t need to add the item marker nor the link to the PR. They are done automatically when rendered.